articles/connections/database/custom-db/create-db-connection.md

```diff
@@ -0,0 +1,152 @@
+---
+description: Learn how to create a database connection.
+toc: true
+topics:
+    - connections
+    - custom-database
+contentType: how-to
+useCase: customize-connections
+---
+Create an Auth0 database connection
+
+The first thing you will do is create a database connection in Auth0:
```

Comment:
Feels like we should have an intro here explaining what the doc is about. If someone lands her directly, and not through the index, they see this and might wonder what's this first thing is about.